scripts/ssm-build.sh

```diff
@@ -210,5 +210,8 @@ then
     lintian "$BUILD_DIR"/apel-ssm-service_"${TAG}"_all.deb
 else
     # Check for errors in SPEC and built packages
-    rpmlint ~/rpmbuild
+    # For instance; Given $(dirname /root/rpmb/rpmbuild/source) will output "/root/rpmb/rpmbuild".
+    RPMDIR=$(dirname $SOURCE_DIR)
+
+    rpmlint $RPMDIR
```

Comment:
Don't really need to split this into two lines.

Reply:
Updated in a0718d8497b608e0f560fc3b2c8b2fc53ae3cda4. Done